### ISSUE TITLE
datacats paster containers with `-d` option should also auto-restart?

### ISSUE THREAD

**Issue by deniszgonjanin** [datacats/datacats#271]
The user probably wants these containers to run in the background and not have to worry about them. If something fails these containers should restart automatically.

Discuss

**Comment by JackMc** [comment]
Easiest way to get this to work would be to have an entrypoint script that kept it alive

**Comment by deniszgonjanin** [comment]
@JackMc Docker itself has a mechanism for this - it's just a flag passed on `docker run` I think.

What I'm curious about is whether this is always a good idea. Might a user want to run a container in the background and NOT have it restart automatically? I can't think of any scenarios where this is the case

**Comment by JackMc** [comment]
Some command which does something important on startup? I can't think of a specific example

**Comment by deniszgonjanin** [comment]
Alternative would be to not have this coupled to the `-d` flag, and have a separate flag instead

**Comment by JackMc** [comment]
Yes, that's what I was thinking. Should it be on or off by default, though?

**Comment by deniszgonjanin** [comment]
Ooh, great question. Off by default would avoid unpleasant surprises?

**Comment by JackMc** [comment]
Yup, principle of latest surprise - I agree
